<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-M23QUF50" failures="1" tests="5" name="Jupiter Cloud tests" time="30.165" errors="0" timestamp="2021-04-27T12:47:59 BST">
  <testcase classname="com.qa.jupitertests.HomeTest" name="clickOnContactsLinkTest" time="4.285"/>
  <testcase classname="com.qa.jupitertests.HomeTest" name="clickOnSubmitFormTest" time="2.288"/>
  <testcase classname="com.qa.jupitertests.HomeTest" name="verifyAlert" time="12.932">
    <failure type="java.lang.AssertionError" message="expected [Thanks fname, we appreciate your feedback.] but found [Thanks mukta, we appreciate your feedback.]">
      <![CDATA[java.lang.AssertionError: expected [Thanks fname, we appreciate your feedback.] but found [Thanks mukta, we appreciate your feedback.]
at com.qa.jupitertests.HomeTest.verifyAlert(HomeTest.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyAlert -->
  <testcase classname="com.qa.jupitertests.HomeTest" name="clickOnShopLinkTest" time="6.277"/>
  <testcase classname="com.qa.jupitertests.HomeTest" name="verifyCartIsShowingSelectedItems" time="0.194"/>
</testsuite> <!-- Jupiter Cloud tests -->
